index.js

```diff
@@ -32,6 +33,14 @@ module.exports = function (options, logger) {
     obscureHeaders = false
   }
 
+  if (excludeHeaders && excludeHeaders.length) {
+    excludeHeaders = excludeHeaders.map(function (name) {
+      return name.toLowerCase()
+    })
+  } else {
+    excludeHeaders = false
+  }
+
```

Comment:
Just a suggestion, maybe we should split this out to a function so we don't repeat the code twice?